python/pipeline/utils/galvo_corrections.py

```diff
@@ -0,0 +1,61 @@
+from scipy.interpolate import interp1d, interp2d
+import numpy as np
+
+
+def correct_motion(img, xymotion):
+    assert isinstance(img, np.ndarray) and len(xymotion) == 2, 'Cannot correct stacks. Only 2D images please.'
+    sz = img.shape
+    y1, x1 = np.ogrid[0: sz[0], 0: sz[1]]
+    y2, x2 = [np.arange(sz[0]) + xymotion[1], np.arange(sz[1]) + xymotion[0]]
+
+    interp = interp2d(x1, y1, img, kind='cubic')
+    img = interp(x2, y2)
+
+    return img
+
+
+def correct_raster(img, raster_phase, fill_fraction ):
+    img = np.array(img)
+    assert img.ndim <= 5, 'Image size greater than 5D.'
+    ix = np.arange(-img.shape[1]/2 + 0.5, img.shape[1]/2 + 0.5) / (img.shape[1]/2)
+
+    tx = np.arcsin(ix * fill_fraction)
+    for ichannel in range(img.shape[2]):
+        for islice in range(img.shape[3]):
+            for iframe in range(img.shape[4]):
+                im = img[:, :, ichannel, islice, iframe].copy()
+                extrapval = np.mean(im)
+                img[::2, :, ichannel, islice, iframe] = interp1d(ix, im[::2, :], kind='linear', bounds_error=False,
+                                                                  fill_value=extrapval)(np.sin(tx +
+                                                                                        raster_phase)/fill_fraction)
+
+                img[1::2, :, ichannel, islice, iframe] = interp1d(ix, im[1::2, :], kind='linear', bounds_error=-False,
+                                                                  fill_value=extrapval)(np.sin(tx -
+                                                                                        raster_phase)/fill_fraction)
+    return img
+
+def plot_raster():
+
+    import matplotlib.pyplot as plt
+    import seaborn as sns
+    from pipeline import preprocess
+    from tiffreader import TIFFReader
+    reader = TIFFReader('/Users/titan/data/cache/11676_4_00006_00009.tif')
```

Comment:
Usually it's not a good idea to have a hard coded file path like this in a repository intended to run on multiple platforms. Consider moving the file path as an argument to this function `plot_raster`.